fuel_utility.py

```diff
@@ -77,7 +77,7 @@ def downloadAndExtractModels(self, directory):
                 logging.info("Model %s already downloaded.", model['name'])
             else:
                 logging.info("Downloading %s", model['name'])
-                url = "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/"+model['name']+"/1/"+model['name']+".zip"
+                url = "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/"+model['name']+"/2/"+model['name']+".zip"
                 response = requests.get(url, stream=True)
```

Comment:
Consider to extract and define these strings in a constant file.
e.g 
OpenRoboticsModelUlr =  "https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models"
ModelVersion = 2
Consider to add method to build this url
e.g 
def BuildModelUrl(modelName):

Reply:
Yep this would be much cleaner to maintain.

The script has a global variable for the fuel server URI but I am not sure what the "2" stands for really.. ignition actually recommends using ign-fuel-tools for fetching models and this best practice will be tracked as a separate issue.. hence I wouldn't go into creating a separate config file for the model URI until the best practices issue is resolved